### PR TITLE
update environment for molecule

### PR DESCRIPTION
**Pull request by kayiwa**
update python, molecule and ansible

### CODE REVIEW

File: .github/workflows/molecule_tests.yml [hunk 1]
```diff
@@ -74,6 +62,14 @@ jobs:
           - pulibrary.apache_maven
           - pulibrary.pulmap
           - pulibrary.memcached
+  test-molecule3:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        role:
+          - figgy
```
Comment (carolyncole) [inline .github/workflows/molecule_tests.yml:70]: I would expect redis, ruby, subversion, vips, nodejs, bind9, freetds, passenger, &  rails-app all to be here...  Was there a reason for no moving them?

File: Pipfile [hunk 2]
```diff
@@ -7,15 +7,14 @@ verify_ssl = true
...
-ansible = "*"
-docker = "*"
 invoke = "*"
 neovim = "*"
 gitpython = "*"
 yamllint = "*"
 flake8 = "*"
 ansible-lint = "*"
+ansible = "*"
+molecule = {extras = ["docker"], version = "*"}
```
Comment (carolyncole) [inline Pipfile:17]: Cool!  This is how you got past the error I was seeing...  Nice!

File: roles/common/.ansible-lint [hunk 3]
```diff
@@ -1,2 +1,3 @@
 skip_list:
   - '303'
+  - '106'
```
Comment (carolyncole) [inline roles/common/.ansible-lint:3]: I know I added these, but I wonder if we should remove them?

**Review by carolyncole — CHANGES_REQUESTED** [review]
Just a couple questions on the git action config and if we should keep all the 106 skips.  We can merge this PR and create tickets to remove those also.

File: .github/workflows/molecule_tests.yml (hunk 1 above)
Comment (kayiwa) [reply .github/workflows/molecule_tests.yml:70]: done.

**Pull request closed without merging**